allofplos/article_class.py

```diff
@@ -53,7 +53,7 @@ def reset_memoized_attrs(self):
         self._tree = None
         self._local = None
         self._contributors = None
-        self._correct_or_retract = None  # Will probably need to be an article subclass
+        self._amendment = None  # Will probably need to be an article subclass
```

Comment:
Why would this need to be an article subclass? What is the effective meaning of this private variable?